lib/cli/interactive-setup/service.js

```diff
@@ -130,22 +151,51 @@ module.exports = {
         projectDir,
         projectName,
       });
-    } else {
-      const projectType = await projectTypeChoice();
-      if (projectType === 'other') {
-        process.stdout.write(
-          '\nRun “serverless create --help” to view available templates and create a new project ' +
-            'from one of those templates.\n'
+    } else if (context.options['template-url']) {
+      projectName = await resolveProjectNameInput(context.options, workingDir);
+      projectDir = join(workingDir, projectName);
+      const templateUrl = context.options['template-url'];
+      try {
+        process.stdout.write(`\n${`Downloading template from provided url: ${templateUrl}...`}\n`);
+        await downloadTemplateFromRepo(templateUrl, null, projectName, { silent: true });
+      } catch (err) {
+        if (err.constructor.name !== 'ServerlessError') throw err;
+
+        throw new ServerlessError(
+          `Could not download template from provided url. Ensure that the template provided with "--template-url" exists. Original error message: ${err.message}`,
```

Comment:
I would strip _. Original error message_ part. I think putting `: <errorMessage>` right after `exists`, will in the end make meaningful error message.